heft-plugins/heft-webpack5-plugin/src/shared.ts

```diff
@@ -23,7 +23,19 @@ export type IWebpackConfiguration =
 /**
  * @public
  */
-export interface IWebpackBundleSubstageProperties extends IBundleSubstageProperties {
+export interface IWebpackVersions {
+  webpackVersion: string;
+  webpackDevServerVersion: string;
+}
+
+/**
+ * @public
+ */
+export interface IWebpackPluginAccessor {
+  /**
+   * A hook that provides the versions of Webpack and Webpack Dev Server.
+   */
+  readonly onEmitWebpackVersionsHook?: AsyncParallelHook<IWebpackVersions>;
```

Comment:
"Emit" is a weird name here.

Why not just include these versions as properties directly on the accessor object?

Reply:
Reason for not including them as properties is because we'd have to load and obtain the versions at `apply` time in order to set them as properties.

Reply:
I just eliminated this now that we're on a common version of webpack-dev-server